site/app/controllers/EmailStatusController.php

```diff
@@ -5,20 +5,19 @@
 use app\libraries\Core;
 use app\libraries\response\MultiResponse;
 use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
 use Symfony\Component\Routing\Annotation\Route;
 use app\views\email\EmailStatusView;
 use app\libraries\routers\AccessControl;
 
-/**
- * @AccessControl(role="INSTRUCTOR")
- */
 class EmailStatusController extends AbstractController {
     public function __construct(Core $core) {
         parent::__construct($core);
     }
 
     /**
      * @Route("/courses/{_semester}/{_course}/email_status")
+     * @AccessControl(role="INSTRUCTOR")
```

Comment:
Why move this from being a class method to a function method?

Reply:
I originally wrote the thread query function in this controller but had a different access control for that function and forgot to change this back to being a class method when I moved the function, I will change it back.